dashboard/api_edx_cache.py

```diff
@@ -74,6 +74,10 @@ class CachedEdxDataApi:
     # the sorting of the supported caches matters for refresh
     SUPPORTED_CACHES = (ENROLLMENT, CERTIFICATE, CURRENT_GRADE,)
     MITXONLINE_SUPPORTED_CACHES = (ENROLLMENT_MITXONLINE, CURRENT_GRADE_MITXONLINE,)
```

Comment:
I think this needs to be this:
```suggestion
    MITXONLINE_SUPPORTED_CACHES = (ENROLLMENT, CURRENT_GRADE,)
```

because otherwise I'm getting this error:

```
web_1      | [2021-09-02 15:02:47] ERROR 14 [dashboard.api] api.py:784 - [b9898e1cf8e7] - Impossible to refresh edX cache
web_1      | Traceback (most recent call last):
web_1      |   File "./dashboard/api.py", line 779, in update_cache_for_backend
web_1      |     CachedEdxDataApi.update_cache_if_expired(user, edx_client, cache_type, provider)
web_1      |   File "./dashboard/api_edx_cache.py", line 344, in update_cache_if_expired
web_1      |     raise ValueError("{} is an unsupported cache type".format(cache_type))
web_1      | ValueError: enrollment_mitxonline is an unsupported cache type
```